backend/app/tests/test_services/test_vault_objective_assignment.py

```
@@ -0,0 +1,140 @@
+        links = (
+            (
+                await async_session.execute(
+                    select(VaultObjectiveProgressLink).where(VaultObjectiveProgressLink.vault_id == vault.id)
+                )
+            )
+            .scalars()
+            .all()
+        )
+
+        assert len(links) == 2
+        linked_ids = {l.objective_id for l in links}
```

Ruff (0.15.17)

[error] Ambiguous variable name: `l`

(E741)